packages/coding-components/templates/angular/angular.json

```diff
@@ -18,7 +18,7 @@
             "main": "src/main.ts",
             "polyfills": ["zone.js"],
```

Comment:
I'm adding the zone.js monkey patch hack to the jsapi-resources Angular sample at 4.29: https://github.com/Esri/jsapi-resources/tree/main/esm-samples/jsapi-angular-cli#known-issues and here's the commit: https://github.com/Esri/jsapi-resources/commit/43ba834d95905a50301ce505ac1bf38b4b953ce3. 

The reason for this is a number of API widgets are broken due to a known bug in Calcite including zoom and editor. And, we didn't want to publish a sample that didn't work with two of the most popular widgets.

Reply:
So does it feel appropriate to add it here? seems like a yes to me?